### ISSUE TITLE
Cloud DNS migration for KCC GKE cluster in sept 2023

### ISSUE THREAD

**Issue by obriensystems** [GoogleCloudPlatform/pubsec-declarative-toolkit#439]
during https://github.com/GoogleCloudPlatform/pubsec-declarative-toolkit/issues/296#issuecomment-1663196129

Starting from September 2023, all GKE Autopilot clusters (minimum version to be determined in August 2023) will migrate to Cloud DNS as the default DNS provider (in replacement of kube-dns), at no extra charge.

However, if you plan to deploy or migrate GKE Autopilot clusters to an [Assured Workloads ](https://cloud.google.com/assured-workloads/docs/concept-key-concepts#folders) folder with an [Impact Level 4 (IL4) ](https://cloud.google.com/security/compliance/disa) compliance regime, please reach out to your account manager to get more details about your migration path.

Learn more about Cloud DNS for GKE
https://cloud.google.com/kubernetes-engine/docs/how-to/cloud-dns?_ga=2.151718137.-1143133292.1680630953

**Comment by cartyc** [comment]
I don't believe this should have any impact on the LZ Config Controller as we're not deploying any workloads or services. Are there any particular areas where you have concerns about how this might impact things?
